lib/resty/upstream/healthcheck.lua

```diff
@@ -201,8 +199,14 @@ local function peer_ok(ctx, is_backup, id, peer)
     end
 end
 
+-- shortcut error function for check_peer()
+local function report_error(sock, ctx, is_backup, id, peer, ...)
+  if not peer.down then errlog(...) end
+  sock:close()
```

Comment:
Yes, not such a big problem, but I personally think it is quite clear that changed.